### PR TITLE
data: SamMintah

### PR DESCRIPTION
**Pull request by SamMintah**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4864"></a>

### CODE REVIEW

File: data/samMintah.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Samuel Mintah",
+  "type": "personal",
+  "bio": "Software Engineer, Tech junkie, early adopter, and futurist. #AI #ML #gadgetlove #innovation",
+  "avatar": "https://github.com/SamMintah.png",
```
Comment (imbeshat) [inline data/samMintah.json:5]:
```suggestion
  
```
It is not needed now

**Review by imbeshat — CHANGES_REQUESTED** [review]

File: data/samMintah.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Samuel Mintah",
+  "type": "personal",
+  "bio": "Software Engineer, Tech junkie, early adopter, and futurist. #AI #ML #gadgetlove #innovation",
```
Comment (imbeshat) [inline data/samMintah.json:4]: If #AI #ML ... are tags that you want to add then you can add them separately.
See this, https://linkfree.eddiehub.io/docs/how-to-guides/tags
If it's part of the bio then it looks good.

File: data/samMintah.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Samuel Mintah",
+  "type": "personal",
+  "bio": "Software Engineer, Tech junkie, early adopter, and futurist.",
+  
+  "tags": [ "Open Source", "Javascript", "Node", "NextJS", "Python",
+   "Rect","full-stack developer"]
```
Comment (imbeshat) [inline data/samMintah.json:7]:
```suggestion
   "React", "Full-Stack Developer"],
```

**Review by imbeshat — CHANGES_REQUESTED** [review]

**Review by imbeshat — DISMISSED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**